README.md

```diff
@@ -20,9 +20,35 @@ BRANCH_NAME=master
 curl --location --remote-name https://github.com/sgmap/mes-aides-ops/archive/$BRANCH_NAME.tar.gz
 tar -xvf $BRANCH_NAME.tar.gz
 cd mes-aides-ops-$BRANCH_NAME
-./bootstrap.sh
+# ./bootstrap.sh mes-aides-ui_head mes-aides-ops_head
+./bootstrap.sh origin/master origin/$BRANCH_NAME
 ```
 
+`./bootstrap.sh origin/master origin/$BRANCH_NAME` initiates the set-up. By default, `origin/master` is used for both repositories (https://github.com/sgmap/mes-aides-ui and https://github.com/sgmap/mes-aides-ops). That can be overidden by passing *TREEISH* parameters. The fist one is `mes-aides-ui` head and the second one is `mes-aides-ops` one.
```

Comment:
- fist → first
- Perhaps italic is enough for _treeish_? :wink:
- I don't understand what you mean by “head”. If I pass a treeish, then it's not necessarily `HEAD`.